### ISSUE TITLE
FluentLenium website is down

### ISSUE THREAD

**Issue by whitejeffreyl** [FluentLenium/FluentLenium#679]
FluentLenium website is down
![screen shot 2019-01-08 at 9 42 59 am](https://user-images.githubusercontent.com/4417751/50837535-dc2c7a00-1329-11e9-8caa-cc66b4cbfdda.png)

**Comment by filipcynarski** [comment]
hello @whitejeffreyl FluentLenium site has been moved to .com domain. http://fluentlenium.com
.org domain belongs to @MathildeLemee and she wasn't been able to set redirect and we decided to move to .com domain because of better control and renewals sorry for inconvenience but we are trying to organize all of the stuff around project to be better manageable and split responsibility to all members of the project  and this was a bottle neck. The information about site change was updated on github project we are aware many people may be confused but we had to make this change and unfortunately it won't be smooth because of lack of response from Mathilde side.
